doc/source/release-notes/1.2.0.rst

```diff
@@ -0,0 +1,40 @@
+.. _PBM-1.2.0:
+
+================================================================================
+*Percona Backup for MongoDB* 1.2.0
+================================================================================
+
+:Date: May 13, 2020
+:Installation: `Installing Percona Backup for MongoDB <https://www.percona.com/doc/percona-backup-mongodb/installation.html>`_
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB
+sharded clusters and replica sets. This is a tool for creating consistent backups
+across a MongoDB sharded cluster (or a single replica set), and for restoring
+those backups to a specific point in time. The project was
```

Comment:
> restoring those backups to a specific point in time

won't it be a confusing statement given we do not support PITR yet?

Reply:
I suppose this text was written by marketing.

Reply:
We can trim this down. Still mentioning mongodb_consistent_backup is outdated by now too. I'll do it.